python-type-hints-multiple-types/README.md

```diff
@@ -0,0 +1,40 @@
+
+# How to Use Type Hints for Multiple Return Types in Python
+
+This folder contains sample code for the [How to Use Type Hints for Multiple Return Types in Python](https://realpython.com/python-type-hints-multiple-types/) tutorial on Real Python.
+
+## Installation
+
+Some of the code requires the following third-party libraries:
+
+- [`mypy`](https://pypi.org/project/mypy/)
+
+To install them into a [virtual environment](https://realpython.com/python-virtual-environments-a-primer/), type the following commands:
+
+```shell
+$ python3 -m venv venv/
+$ source venv/bin/activate
+(venv) $ python -m pip install -r requirements.txt
+```
+
+## Running
+
+### 01_type_hints_for_one_piece_of_data
+
+
+### 02_type_hints_for_multiple_pieces_of_data
+
+
+### 03_type_hints_for_callback
+
+
+### 04_type_hints_for_factory_function
+
+
+### 05_type_hints_for_generator
+
+
+### 06_type_aliases
+
+
+### 07_static_type_checking
```

Comment:
If you're unsure what to put in the README file, then I'd say let's cut the highlighted lines.